Sources/StreamChatUI/ChatChannelList/Search/ChatMessageSearchVC.swift

```diff
@@ -105,6 +109,10 @@ open class ChatMessageSearchVC: ChatChannelListSearchVC, ChatMessageSearchContro
 
     // MARK: - ChatMessageSearchControllerDelegate
 
+    override open func controller(_ controller: DataController, didChangeState state: DataController.State) {
+        reloadMessages()
+    }
```

Comment:
wait, this will break the state logic right? So, the loading views etc won't appear?

Reply:
I added `didSet` to `messages` which then calls the validation.